frontend/app/documents/page.tsx

```
@@ -39,6 +39,7 @@ import type { RouterOutput } from "@/trpc";
 import { DocumentTemplateType, DocumentType, trpc } from "@/trpc/client";
 import { assertDefined } from "@/utils/assert";
 import { formatDate } from "@/utils/time";
+import { FinishOnboarding } from "@/app/documents/FinishOnboarding";
```

ESLint

[error] `@/app/documents/FinishOnboarding` import should occur before import of `@/components/DataTable`

(import/order)